external/builder/test-fixtures_esprima.js

```diff
@@ -35,20 +38,22 @@ ls('*-expected.*').forEach(function(expectationFilename) {
   if (out !== expectation) {
     errors++;
 
-    echo('Assertion failed for ' + inFilename);
-    echo('--------------------------------------------------');
-    echo('EXPECTED:');
-    echo(expectation);
-    echo('--------------------------------------------------');
-    echo('ACTUAL');
-    echo(out);
-    echo('--------------------------------------------------');
-    echo();
+    console.log('Assertion failed for ' + inFilename);
+    console.log('--------------------------------------------------');
+    console.log('EXPECTED:');
+    console.log(expectation);
+    console.log('--------------------------------------------------');
+    console.log('ACTUAL');
+    console.log(out);
+    console.log('--------------------------------------------------');
+    console.log();
   }
 });
 
 if (errors) {
-  echo('Found ' + errors + ' expectation failures.');
+  console.error('Found ' + errors + ' expectation failures.');
+  process.exit(1);
```

Comment:
Shouldn't `external/builder/test-fixtures.js` have this as well, since they're quite similar? The same holds for the `else` branch here.